src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+enum Color {
+    BLUE,
+    RED,
+    GREEN
+}
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return number + ", " + color;
```

Comment:
When overriding `toString()`, it's better to include the class name and field names for clarity. This can be particularly helpful during debugging.